docs/src/modules/java/pages/views.adoc

```diff
@@ -180,13 +180,13 @@ The source of a View can be a topic. It works the same way as shown in <<event-s
 NOTE: For the messages to be correctly consumed in the view, there must be a `ce-subject` metadata associated with each message. This is required because for each message consumed from the topic there will be a corresponding view row. That view row is selected based on such `ce-subject`. See the example below for how to pass such metadata when producing to a topic with Akka SDK.
 
 [source,java]
-.{sample-base-url}/event-sourced-counter-brokers/src/main/java/counter/application/CounterJournalToTopicWithMetaConsumer.java[CounterJournalToTopicWithMetaConsumer.java]
+.{sample-base-url}/event-sourced-counter-brokers/src/main/java/counter/application/CounterTopicView.java[CounterTopicView.java]
 ----
-include::example$event-sourced-counter-brokers/src/main/java/counter/application/CounterJournalToTopicWithMetaConsumer.java[tag=class]
+include::example$event-sourced-counter-brokers/src/main/java/counter/application/CounterTopicView.java[tag=class]
 ----
 <1> Uses `@Consume.FromTopic` annotation to set the target topic.
-<2> Composes the `ce-subject` attribute from the entity's event subject.
-<3> Returns a producing effect with the updated metadata together with the message payload.
+<2> Composes the `ce-subject` attribute from the entity's event subject to include in the view row.
```

Comment:
that might not be available in the topic data, maybe we could mention about that?

Reply:
There is a NOTE section just above the snippet about it being required, so I think that is enough